public/locales/en/panel-nightsky.json

```
@@ -0,0 +1,249 @@
+{
+  "tab-headers": {
+    "markings": "Markings",
+    "time": "Time",
+    "location": "Location / View",
+    "stars": "Stars",
+    "solar-system": "Solar System",
+    "sun": "Sun"
+  },
+  "markings": {
+    "button-labels": {
+      "hide-all": "Hide All",
+      "grid": "Grid",
+      "labels": "Labels",
+      "line_one": "Line",
+      "line_other": "Lines"
+    },
+    "alt-az": {
+      "title": "Alt/Az",
+      "buttons": {
+        "meridian": "Meridian",
+        "zenith": "Zenith"
+      }
+    },
+    "ecliptic": {
+      "title": "Ecliptic",
+      "buttons": {
+        "band": "Band"
+      }
+    },
+    "equitorial": {
+      "title": "Equitorial"
+    },
+    "constellations": {
+      "title": "Constellations",
+      "buttons": {
+        "art": "Art"
+      },
+      "aria-labels": {
+        "lines": "Show constellation lines",
+        "labels": "Show constellation labels",
+        "art": "Show constellation art"
+      }
+    },
+    "cardinal-directions": {
+      "title": "Cardinal Directions",
+      "buttons": {
+        "small": "Small",
+        "large": "Large",
+        "marks": "Marks"
+      },
+      "aria-labels": {
+        "small": "Show small cardinal directions",
+        "large": "Show large cardianl directions",
+        "marks": "Show marks on cardinal directions"
+      }
+    }
+  },
+  "location": {
+    "globe-location": {
+      "title": "Globe Location",
+      "latitude": "Latitude",
+      "longitude": "Longitude",
+      "altitude": "Altitude",
+      "meter-abbreviation": "m",
+      "warning": {
+        "title": "Warning!",
+        "description": "You are not on Earth, but on {{name}}."
+      }
+    },
+    "jump-to-position": {
+      "title": "Jump to Position",
+      "north-pole": "North Pole",
+      "equator": "Equator",
+      "south-pole": "South Pole",
+      "tooltip": "To search for a specific location on Earth, use the Geo Location window."
+    },
+    "direction": {
+      "title": "Direction",
+      "look-north": "Look North",
+      "look-east": "Look East",
+      "look-south": "Look South",
+      "look-west": "Look West"
+    },
+    "horizon": {
+      "title": "Horizon",
+      "look-horizion": "Look at Horizon",
+      "level-horizon": "Level Horizon",
+      "look-up": "Look Up"
+    }
+  },
+  "solar-system": {
+    "trails": {
+      "title": "Trails",
+      "buttons": {
+        "hide-trails": "Hide ALL Trails",
```

```suggestion
        "hide-trails": "Hide all Trails",
```

For styling consistency